worlds/musedash/docs/setup_en.md

```diff
@@ -0,0 +1,36 @@
+# Muse Dash Randomizer Setup Guide
+
+## Quick Links
+- [Main Page](../../../../games/Muse%20Dash/info/en)
+- [Settings Page](../../../../games/Muse%20Dash/player-settings)
+
+## Required Software
+
+- Windows 8 or Newer.
+- Muse Dash: [Available on Steam](https://store.steampowered.com/app/774171/Muse_Dash/)
+  - \[Optional\] Just As Planned DLC: [Also Available on Steam](https://store.steampowered.com/app/1055810/Muse_Dash__Just_as_planned/)
+- Melon Loader v0.6.1: [GitHub](https://github.com/LavaGang/MelonLoader/releases/tag/v0.6.1) (Note: v0.5.7 or older is not compatible with this mod.)
+- Muse Dash Archipelago Mod: [GitHub](https://github.com/DeamonHunter/ArchipelagoMuseDash/releases/latest)
+
+## Installing the Archipelago mod to Muse Dash
+
+1. Download [MelonLoader.Installer.exe](https://github.com/LavaGang/MelonLoader/releases/tag/v0.6.1) and run it.
```

Comment:
It might be a better idea to link to https://github.com/LavaGang/MelonLoader/releases or https://github.com/LavaGang/MelonLoader/releases/latest instead of a specific tag which may eventually be out of date. Or if for some reason v0.6.1 is specifically the version of the loader we want to use and not newer, that should be mentioned in the text.

Reply:
I've updated all the links to /latest. Should definitely help with confusion.